AsyncDisplayKit.xcodeproj/project.pbxproj

```diff
@@ -1874,12 +1884,12 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 058D09D2195D04C000B7D73C /* Build configuration list for PBXNativeTarget "AsyncDisplayKitTests" */;
 			buildPhases = (
-				2E61B6A0DB0F436A9DDBE86F /* 📦 Check Pods Manifest.lock */,
+				2E61B6A0DB0F436A9DDBE86F /* [CP] Check Pods Manifest.lock */,
 				058D09B8195D04C000B7D73C /* Sources */,
 				058D09B9195D04C000B7D73C /* Frameworks */,
 				058D09BA195D04C000B7D73C /* Resources */,
-				3B9D88CDF51B429C8409E4B6 /* 📦 Copy Pods Resources */,
-				B130AB1AC0A1E5162E211C19 /* 📦 Embed Pods Frameworks */,
+				3B9D88CDF51B429C8409E4B6 /* [CP] Copy Pods Resources */,
+				B130AB1AC0A1E5162E211C19 /* [CP] Embed Pods Frameworks */,
```

Comment:
Do we need this changes here? Which Cocoapods version are you using, I think with 1.0 they changed that ...